api/.env.example

```diff
@@ -31,5 +29,5 @@ DATACITE_ENDPOINT=https://api.test.datacite.org/dois
 DATACITE_USER=user
 DATACITE_PASSWORD=password
 
-MAIL_SERVER=0.0.0.0
-LOCALSTACK_SERVER=localhost
\ No newline at end of file
+MAIL_SERVER=localhost
```

Comment:
Is there a reason for changing MAIL_SERVER? This is breaking the mailing service for me